### ISSUE TITLE
Span API protected?

### ISSUE THREAD

**Issue by jeffkibuule** [galak/span-hacs#7]
Is anyone seeing denied access to the SPAN API? I'm running `spanos2/r202240/06` and am not getting any data as of a day ago. Instead when I try to access `/api/v1/circuits/`, I get the following back:

`{"detail":"Access to protected HTTP endpoints require an API token passed as a HTTP Bearer header."}`

**Comment by s0schen** [comment]
I was just updated to spanos2/r202240/06.1   I notice the webpage access is now password protected with a password I don't have.    The webpage allows you to hit the door button 3 times to disable the password and to log in.   After log in everything looks the same.

**Comment by wez** [comment]
Yeah, I'm seeing this error as well. I've reached out to span to see if they'll share info on how to authenticate this.
I also just pushed a fix for a missing exception import so that the integration handles this issue slightly better.

**Comment by wez** [comment]
I stumbled across this gist which has some API related info, including stuff about auth:
https://gist.github.com/hyun007/c689fbed10424b558f140c54851659e3
